src/main/resources/db/migration/V1__Create_initial_tables.sql

```diff
@@ -0,0 +1,5 @@
+CREATE TABLE cards (
+  id BIGSERIAL PRIMARY KEY,
+  document JSONB
+);
```

Comment:
Her kommer vi også til å trenge kolonnen `revision` for å støtte optimistisk låsing.